k8s/upgrades/0.8.0-0.8.1/cstor_volume_update.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+set -e
+
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+function usage() {
+    echo 
+    echo "Usage:"
+    echo 
+    echo "$0 <pv-name> <openebs-namespace>"
+    echo 
+    echo "  <pv-name> Get the PV name using: kubectl get pv"
+    echo "  <openebs-namespace> Get the namespace where openebs"
+    echo "  pods are installed"
+    exit 1
+}
+
+function setDeploymentRecreateStrategy() {
+    dns=$1 # deployment namespace
+    dn=$2  # deployment name
+    currStrategy=`kubectl get deploy -n $dns $dn -o jsonpath="{.spec.strategy.type}"`
+
+    if [ $currStrategy = "RollingUpdate" ]; then
```

Comment:
better to check for 'Recreate' rather than 'RollingUpdate'

Reply:
for newly created volumes, strategy should be Recreate

Reply:
volumes provisioned in 0.8.0 will not be having it so we need this check